recipe/meta.yaml

```diff
@@ -19,32 +17,42 @@ build:
   skip: true  # [win and vc<14]
   missing_dso_whitelist:  # [linux and s390x]
     - $RPATH/ld64.so.1    # [linux and s390x]
+  script_env:
+    # "Environment variable VCPKG_FORCE_SYSTEM_BINARIES must be set on
+    # arm, s390x, ppc64le and riscv platforms."
+    - VCPKG_FORCE_SYSTEM_BINARIES=1  # [ arm64 or aarch64 or s390x or ppc64le]
+  overlinking_ignore_patterns:
+    # complains about libfmt.so
+    - "lib/libtiledb.*"
+
 requirements:
   build:
-    - make  # [not win]
-    - git   # [not win]
-    - sed   # [not win]
-    - gawk  # [not win]
-    - cmake <=3.21
-    - pkg-config  # [not win]
+    - ninja
+    - sed                    # [not win]
+    - gawk                   # [not win]
+    - cmake                  # >=3.21
+    - pkg-config             # [not win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch  # [not win]
-    - m2-patch  # [win]
   host:
     - zlib {{ zlib }}
     - bzip2 {{ bzip2 }}
     - zstd {{ zstd }}
-    - lz4-c 1.9.4
-    - curl 7.88.1 # [not win]
-    - libcurl 7.88.1  # [not win]
-    - openssl  {{ openssl }}  # [not win]
+    - lz4-c
+    - libcurl
+    - openssl {{ openssl }}
+    - spdlog
+    - capnproto 1.0.1
+    - libmagic  # [not win]
+    # Git is required for vcpkg
+    # Ideally, we'd unvendor vcpkg as mentioned in build.bat
+    # But that would require building libmagic (aka file) on windows
+    - git  # [win]
   run:
+    - fmt  # [win]
     - openssl 3.*
 
 test:
-  #  downstreams:
-  #  - tiledb-py
   commands:
     - test -e $PREFIX/include/tiledb/tiledb.h  # [not win]
     - test -e $PREFIX/include/tiledb/tiledb  # [not win]
```

Comment:
Could you please also check `Library/lib/pkgconfig/tiledb.pc` on `win` and `lib/pkgconfig/tiledb.pc` on `unix`?

Reply:
done